securedrop/source_app/session_manager.py

```diff
@@ -30,7 +30,7 @@ class SessionManager:
     """Helper to manage the user's session cookie accessible via flask.session."""
 
     # The keys in flask.session for the user's passphrase and expiration date
-    _SESSION_COOKIE_KEY_FOR_CODENAME = "codename"
+    _SESSION_COOKIE_KEY_FOR_CODENAME = "passphrase"
```

Comment:
Should be _SESSION_COOKIE_KEY_FOR_PASSPHRASE here. Nice rename btw 👍

Reply:
Turns out the only thing in computer science more annoying than naming things is renaming them :)